src/test/java/foodwhere/logic/commands/EditCommandTest.java

```diff
@@ -19,7 +19,7 @@
 import foodwhere.testutil.TypicalStalls;
 
 /**
- * Contains integration tests (interaction with the Model) and unit tests for EditCommand.
+ * Contains integration tests (interaction with the Model) and unit tests for SEditCommand.
  */
 public class EditCommandTest {
```

Comment:
Perhaps this should be renamed SEditCommandTest so we can split sedit and redit?

Reply:
Fixed and resolved in the latest push. Thank you for spotting it.